docs/commands/logout.md

```diff
@@ -0,0 +1,35 @@
+# smctl version
+
+## Overview
+`smctl logout` allows to logout from service manager.
```

Comment:
Log off Service Management

Reply:
smctl is still used in the opensource, there we are still service manager. preheps not to include the name at all?